src/screens/WalletConnectApprovalSheet.js

```diff
@@ -37,23 +48,70 @@ const DappLogo = styled(RequestVendorLogoIcon).attrs(
   margin-bottom: 24;
 `;
 
+const NetworkLabelText = styled(Text).attrs(({ theme: { colors } }) => ({
+  color: colors.blueGreyDark,
+  lineHeight: 17,
+  size: 'lmedium',
+}))`
+  margin-bottom: 4;
+`;
+
+const LabelText = styled(Text).attrs(() => ({
+  lineHeight: 22,
+  size: 'large',
+  weight: 'heavy',
+}))``;
+
+const AvatarWrapper = styled(View).attrs(() => ({}))`
```

Comment:
If you're not using `attrs` there's no need to declare it. You can do this:

```suggestion
const AvatarWrapper = styled(View)`
```